torch/nn/functional.py

```diff
@@ -2133,7 +2129,7 @@ def embedding(
     scale_grad_by_freq: bool = False,
     sparse: bool = False,
 ) -> Tensor:
-    r"""A simple lookup table that looks up embeddings in a fixed dictionary and size.
+    r"""Lookup table that looks up embeddings in a fixed dictionary and size.
```

Comment:
In the above phrase, it should be A lookup table (noun). It needs to be rephrased to start with a verb. Perhaps, "Generate a simple lookup table"?